.travis.yml

```
@@ -36,10 +36,13 @@ jobs:
       before_deploy:
         - poetry config repositories.mdsol https://mdsol.jfrog.io/mdsol/api/pypi/pypi-local
         - poetry config http-basic.mdsol $ARTIFACTORY_USERNAME $ARTIFACTORY_PASSWORD # Stored as Travis CI Environment Vars
+        - poetry config pypi-token.pypi $POETRY_PYPI_TOKEN_PYPI  # this may be unnecessary
         - poetry build
       deploy:
         provider: script
-        script: poetry publish -r mdsol
+        script: 
+          poetry publish -r mdsol
+          poetry publish
```

`-` is missing:
```suggestion
        script: 
          - poetry publish -r mdsol
          - poetry publish
```

but if we don't need to publish to Artifactory, it will be just:
```
script: poetry publish
```

---

why do we need the script provider to begin with? travis support PyPI: https://docs.travis-ci.com/user/deployment-v2/providers/pypi/

---

because we are using poetry https://github.com/python-poetry/poetry/issues/669